### ISSUE TITLE
Writing Thanks Card with image

### ISSUE THREAD

**Issue by naotakanishiyama** [givlink/giv-app#17]
Some members cannot write and edit Thanks Card properly which are both for iOS and Android.
(メンバーによってはThanks Cardが適切に記入/編集ができない状況( iOS&Android))

- Click on お知らせ bottom for writing the TC (5 notifications), there is no place for writing. White page is shown.
(5つの通知が来ているメンバーがTCを記入するためにお知らせ欄を開くと、真っ白で何も表示されていないので書けない)

![IMG_4733](https://user-images.githubusercontent.com/60317544/118418530-bb437300-b6f3-11eb-9b8c-b5e5f977a04b.JPG)

- When a member upload image after writing TC, those screens shown (those seem fine).
   But the user select specific image, suddenly back to timeline page.
   （すでに記載済みのTCに対して後から写真を追加するために写真を選択すると、タイムラインに戻ってしまう）
![IMG_4735](https://user-images.githubusercontent.com/60317544/118418535-be3e6380-b6f3-11eb-8774-0cd543b42937.JPG)

![IMG_4734](https://user-images.githubusercontent.com/60317544/118418534-bd0d3680-b6f3-11eb-9c89-9192e659a5f9.JPG)

**Comment by HelloKashif** [comment]
Yes this suddenly going back to timeline page is a known bug in iOS app. This only happens for the first time when selecting images. Second time onwards this works. I'll find root cause but for now please ask them to try one more time. 

As for notifications issue it was fixed in #20

**Comment by naotakanishiyama** [comment]
Thank you Kashif-san, I got it.

**Comment by HelloKashif** [comment]
@naotakanishiyama san fix needed resubmit on Apple. Just submitted it, will take 2-3 days I think.

**Comment by naotakanishiyama** [comment]
@HelloKashif san, Sure. Should I click on 再審査 on Apple now? or please let me know if i can do anything.
